### DISCUSSION TITLE
Is it possible to get the custom webpack module to run before postcss?

### DISCUSSION THREAD

**Discussion by pruge** [vercel/next.js#45416] [Help]
I wrote a webpack module that adds a dark prefix.
This module will behave as follows:
```js
export const Button = () => {
  return (
    <button className={cx({
        base: 'inline-flex items-center rounded',
        light: 'bg-sky-200 text-black',
        dark: 'bg-sky-400 text-white'
    })}>
      hello
    </button>
  )
}

Changes to: 

export const Button = () => {
  return (
    <button className={cx({
        base: 'inline-flex items-center rounded',
        light: 'bg-sky-200 text-black',
        dark: 'dark:bg-sky-400 dark:text-white'  // <-- add dark prefix
    })}>
      hello
    </button>
  )
}

actually :

<div class="dark">
    <button class="inline-flex items-center rounded bg-sky-200 
        text-black dark:bg-sky-400 dark:text-white"
    >
      hello
    </button>
</div>
```

In fact, the class name is well written, but the css of dark:bg-sky-400 and dark:text-white are not attached.,I think my webpack module's sequence of actions is behind the postcss.

Can I specify the order of actions?
```js
// next.config.js
  webpack(config, options) {

    // tailwind, add dark prefix
    config.module.rules.push({
      test: /\.(tsx)$/,
      include: [path.resolve(__dirname, 'src/components')],
      use: ['tailwind-clsx'],
    })

    return config
  },
```

**Comment by icyJoseph** [comment]
I am not sure what is it exactly you are trying to do, but perhaps you can unshift into the rules? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/unshift

I think the PostCSS also runs in the minification step and that might be causing issues. Console log `config.optimization.minimizer`, I think there's two there, one of them does some CSS minification.

**Reply by pruge** [reply]
thank you @icyJoseph
I've tried unshift, but the results are the same. and the issue does not occur in build.
